runtime/libpgmath/lib/aarch64/cos/CMakeLists.txt

```diff
@@ -23,26 +23,28 @@ set(SRCS)
 # Set source files, compiler flags and definitions
 if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64")
```

Comment:
why do we have this x86_64 part in CMakeLists.txt down below aarch64 directory?

Reply:
Good catch, I'll clean that up.